app/uk/gov/hmrc/merchandiseinbaggage/connectors/EoriCheckConnector.scala

```diff
@@ -19,16 +19,19 @@ package uk.gov.hmrc.merchandiseinbaggage.connectors
 import javax.inject.{Inject, Named, Singleton}
 import uk.gov.hmrc.http.HttpReads.Implicits._
 import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
-import uk.gov.hmrc.merchandiseinbaggage.config.EoriCheckConfiguration
+import uk.gov.hmrc.merchandiseinbaggage.config.AppConfig
 import uk.gov.hmrc.merchandiseinbaggage.model.api.Eori
 import uk.gov.hmrc.merchandiseinbaggage.model.api.checkeori.CheckResponse
 
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class EoriCheckConnector @Inject() (httpClient: HttpClient, @Named("eoriCheckBaseUrl") base: String)
-    extends EoriCheckConfiguration {
+class EoriCheckConnector @Inject() (
+  config: AppConfig,
```

Comment:
use `appConfig` instead of config for consistency with the `CurrencyConversionConnector`, thanks.